.github/workflows/docker-image.yml

```diff
@@ -43,8 +43,8 @@ jobs:
         tags: ${{ steps.meta.outputs.tags }}
         labels: ${{ steps.meta.outputs.labels }}
         build-args:  |
-                  UI_TAG=${{ steps.build-args.outputs.KAOTO_TAG }}
-                  API_TAG=${{ steps.build-args.outputs.KAOTO_TAG }}
+                  UI_TAG=main
```

Comment:
This job is for nightly (always main) and PR check (whose branch name does not have a corresponding branch in UI and backend)